toolchain/parse/node_kind.def

```diff
@@ -533,6 +555,7 @@ CARBON_PARSE_NODE_KIND_CHILD_COUNT(IfExprElse, 3,
 // StructLiteral
```

Comment:
It's not clear to me what's being repeated here. I think two things would help:

- Indent the repeated parts.
- Put the `_repeated_` label *after* rather than before, like we do for other parent nodes.

So something like:

```c++
// Struct literals, such as `{.a = 0}`:
//   StructLiteralOrStructTypeLiteralStart
//         _external_: Name
//       StructFieldDesignator
//       _external_: expression
//     StructFieldValue
//     StructComma
//   _repeated_
// StructLiteral
```

(Maybe something more explicit like `_above nodes repeated_` would also make this clearer?)

That said, I'm planning to remove these comments as part of #3393 so maybe this isn't worth worrying about.